omegatrade-app/backend/app/models/user.model.js

```diff
@@ -0,0 +1,37 @@
+'user strict';
+const database = require('./../config/database.js');
+var User = function () {};
+
+User.registerUser = async function (user, cb) {
+  const usersTable = database.table('users');
+  try {
+    await usersTable.insert(user);
+    cb(null, true);
+    return true;
+  } catch (err) {
+    cb(err, null);
+      return;
+  } 
+}
+
+User.findUser = async function (email, cb) {
+  try {
+    const query = {
+      sql: 'SELECT userId,fullName,businessEmail,password,photoUrl,provider FROM users where businessEmail = @businessEmail',
+      params: {
+        businessEmail: email
+      }
+    };
```

Comment:
(Here and elsewhere): You can instruct Spanner to return the result as JSON automatically. That removes the need to manually map each row to JSON:

```suggestion
    const query = {
      sql: 'SELECT userId,fullName,businessEmail,password,photoUrl,provider FROM users where businessEmail = @businessEmail',
      params: {
        businessEmail: email
      },
      json: true,
    };
```

Reply:
removed manual mapping to JSON and added `json : true` in the query to return result as JSON.